src/components/Dissolution/CompleteResolution.vue

```diff
@@ -577,7 +577,9 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
           ]
         }
     }
-    this.setResolutionStepValidationDetail(validationDetail)
+    if (validationDetail) {
```

Comment:
Safety check: we don't want to store null for validation detail; instead, leave the default values.